python/docs/source/conf.py

```diff
@@ -0,0 +1,69 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+from typing import List
+from recommonmark.transform import AutoStructify
+
+# Set python project root
+sys.path.insert(0, os.path.abspath("../../"))
+
+# The master toctree document
+master_doc = "index"
+
+# -- Project information -----------------------------------------------------
+
+project = "ml4ir"
+copyright = "2020, Search Relevance (Salesforce.com, Inc.)"
+author = "Search Relevance (Salesforce.com, Inc.)"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1.0"
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions: List = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.coverage",
+    "sphinx.ext.napoleon",
+    "recommonmark",
+]  # noqa: E501
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path: List = ["_templates"]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns: List = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "default"
+
```

Comment:
you should add `html_title` (with the `version`)

currently, the HTML `title` is

`<title>ml4ir - Machine Learning for Information Retrieval &mdash; ml4ir 0.1.0 documentation</title>
` 

for https://ml4ir.readthedocs.io/en/latest/index.html

and 

`<title>Metrics &mdash; ml4ir 0.1.0 documentation</title>
`

for 

https://ml4ir.readthedocs.io/en/latest/api/metrics.html

Reply:
I will look into it. I didn't know about this.